test/system/helpers.bash

```diff
@@ -336,8 +339,8 @@ function run_podman() {
     # Remember command args, for possible use in later diagnostic messages
     MOST_RECENT_PODMAN_COMMAND="podman $*"
 
-    # stdout is only emitted upon error; this echo is to help a debugger
-    echo "$(timestamp) $_LOG_PROMPT $PODMAN $*"
+    # stdout is only emitted upon error; this printf is to help in debugging
+    printf "\n%s %s %s\n" "$(timestamp)" "$_LOG_PROMPT" "$*"
```

Comment:
This is completely unrelated, I'm just sneaking it in. It adds an extra newline, which has greatly helped me scan error logs for most-recent-podman-command.